Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -111,6 +111,9 @@ public struct IndexOptions: Codable {
     /// Optionally specifies fields in the index and their corresponding weight values.
     public var weights: BSONDocument?
 
+    /// Optionally specifies the wildcard projection of a wildcard index.
+    public var wildcardProjection: BSONDocument?
```

Comment:
Can we update this to be more more specific? Something like "Optionally specifies a projection document used to determine which fields are indexed by a wildcard index."

Can specify that this feature is only supported in MongoDB 4.2+? 

Also, can we add a link / direct users to dochub.mongodb.org/core/index-wildcard for more information about wild card indexes.